src/webchat-ui/components/WebchatUI.tsx

```diff
@@ -314,6 +314,47 @@ export class WebchatUI extends React.PureComponent<React.HTMLProps<HTMLDivElemen
             />
         );
     }
+    
+    handleKeydown = (event) => {
+        const closeButtonInHeader = document.getElementById("webchatHeaderCloseButton");
+        const chatToggleButton = document.getElementById("webChatToggleButton");
+        const webchatHistoryPanel = document.getElementById("webchatChatHistory");
+        const textMessageInput = document.getElementById("webchatTextMessageInput");
+        const getStartedButton = document.getElementById("webchatGetStartedButton");
+        const webchatInputButtonMenu = document.getElementById("webchatInputButtonMenu");
+
+        if(event.target === closeButtonInHeader) {
+            if(event.shiftKey && event.key == "Tab") {
+                event.preventDefault();
+                if(textMessageInput) {
+                    textMessageInput.focus();
+                } else if (getStartedButton) {
+                    getStartedButton.focus();
+                } else if (webchatInputButtonMenu) {
+                    webchatInputButtonMenu.focus();
+                } else {
+                    chatToggleButton?.focus()
+                }
+            }
+        }
+        if(event.target === chatToggleButton) {
```

Comment:
These cases deserve a line of comment or two. What is special about pressing (shift)+tab on these two elements?

Reply:
Added comments